Core/Core/Discussions/GetDiscussions.swift

```diff
@@ -85,3 +85,45 @@ class GetDiscussionView: CollectionUseCase {
         }
     }
 }
+
+class GetDiscussionEntry: GetDiscussionView {
```

Comment:
There is no api to get a single entry, so this requests the whole View, but scopes down to just the specific entry. Since this is used on the reply, it should be rare that the request isn't already cached. (This also inherits the cache key)